swap/src/bitcoin.rs

```diff
@@ -1,26 +1,24 @@
+pub use crate::bitcoin::{
+    timelocks::Timelock,
+    transactions::{TxCancel, TxLock, TxPunish, TxRedeem, TxRefund},
+};
+use crate::{bitcoin::timelocks::BlockHeight, config::Config, ExpiredTimelocks};
 use ::bitcoin::{
     hashes::{hex::ToHex, Hash},
     secp256k1,
     util::psbt::PartiallySignedTransaction,
     SigHash,
 };
+pub use ::bitcoin::{util::amount::Amount, Address, Network, Transaction, Txid};
 use anyhow::{anyhow, bail, Result};
 use async_trait::async_trait;
 use ecdsa_fun::{adaptor::Adaptor, fun::Point, nonce::Deterministic, ECDSA};
+pub use ecdsa_fun::{adaptor::EncryptedSignature, fun::Scalar, Signature};
```

Comment:
I would recommend to keep the `pub use` separated as they often have a different function from `use`.
The `pub use` are about re-exporting other modules/crates items to have them on the API of the given module. Whereas `use` are about importing what is needed inside the module.